### ISSUE TITLE
Wurst for modded clients

### ISSUE THREAD

**Issue by xXxswagtastical69xXx** [Wurst-Imperium/Wurst-Client#777]
Vanilla is overrated. Wurst should be able to work on modded clients too. Please?

Very late edit (6/23/2015): If this is actually going to happen, it needs to be for 1.7.10

**Comment by pokeball99** [comment]
U mean forge mods?I think that's rejected

**Comment by Alexander01998** [comment]
> U mean forge mods?I think that's rejected

Forge + OptiFine is rejected, but Forge isn't.

**Comment by pokeball99** [comment]
I think building forge into wurst would be bad,as it sends info to the server

**Comment by Alexander01998** [comment]
> I think building forge into wurst would be bad,as it sends info to the server

What do you mean by that?

**Comment by pokeball99** [comment]
I don't know the exactly what it is,but forge sends details of mods,plus some other stuff I think,to the server,and they have plugins to detect if u have a certain mod on and can kick,ban or what ever else they want to u

**Comment by xXxswagtastical69xXx** [comment]
I've tried Wurst 1.3.1 with the FTB Trident 1.1.0 modpack by overwriting the "1.7.10" version .jar with the Wurst 1.3.1 client, along with JVM argument -Dfml.ignoreInvalidMinecraftCertificates=true so FML ignores mismatched checksums. The game won't start. 

Just pointing that out.

**Comment by Alexander01998** [comment]
> I don't know the exactly what it is,but forge sends details of mods,plus some other stuff I think,to the server,and they have plugins to detect if u have a certain mod on and can kick,ban or what ever else they want to u

As far as I know, Forge does nothing like that by itself.
However, many mods send custom packets to the server to handle their special features, which a plugin could detect.

> I've tried Wurst 1.3.1 with the FTB Trident 1.1.0 modpack by overwriting the "1.7.10" version .jar with the Wurst 1.3.1 client, along with JVM argument -Dfml.ignoreInvalidMinecraftCertificates=true so FML ignores mismatched checksums. The game won't start.

Wurst for MC 1.7.X is based on Minecraft 1.7.2, not 1.7.10.
But even with Forge for MC 1.7.2, this method wont work, as Wurst and Forge overwrite the same classes.

**Comment by AspireFrostbyte** [comment]
I actually have made this (wurst combined with forge) for myself so I can confirm / inform about a few things.
1 it is possible, I did it by building a forge environment then taking the forge mod loader src code from it and importing it into an mcp environment with wurst then I spent a day or so editing minecraft base classes (and some forge classes) so both forge and wurst where happy.
2 no it did not send any info to the server because the server only sends info about mods loaded, and this was a modified version of forge itself not a mod.
3 I wont be releasing a download link cus 2 reasons 1 alexander prob wont be happy with it, and 2 forge wont be happy with it, if anyone wants proof just ask me and i upload some images to imgur or make a video or somting
it was easy to do just time consuming 90% of it was copy and paste
